mappings/6ce39fbdb3eec8331a7cd7436daa3b0db1e546497c504c531862b9284b4650414e4441.json

```diff
@@ -0,0 +1,64 @@
y8/q3nIjYIJWnIVteBu8J01bjjLFl1r+FJDvXDjgxVZYiOm/f7QjJ58qa35ZhR/noPOqBhoFLGfFTPa+t+9akV62vrGUnpUJrfT5HSYvKmbsg9N+uSAhususoNurie4S/HqvtMT6M1b4LU9PZONZ9AyvEizssks84rwkLoBpap1ijBgEUtmFrRqjQR9HqX0trhFheWygeHcK7X1m0L7XCZiuNTCoYyzJjCzBILb4rH47GdRWDZFjuA1QEmHGoVrZPZKJVyDcIEXU2NlFq5R6NyR1kY4vf9y9QyFJadnruKRV5aX9afF/JW8kN5p5tCdkGVnUnquVB57GPffyLXesMc518kPV/ZB5OMODqUWB/ejfUzr8bpZ54C6TqflJTIQ3NAWH6JiXTO5+VCdO68ObjFSTYc4lOobH2GZY+yMXnRfzK1EO9vNjgbZx6ETkHOgbm7btoR5wyxodZlGpZj2S671SDIKGm03lyH41nSQZ2ks44ynQMtAtljaTQfhGeb8ph/kp4okeE9OkuFpY2UaxBhB3MowY8LNYiG5+bQQsvruYpgYdGVmJpN22I0Ki8tTwoovvwUejhJzdvjVZ/mDD5I4f3U3CSRbp0LlxCl1uWej5u7Zso8/qnx3xw/gtbkO/dSG3sxKyn1lsFWzrmcLbXucpJ8kqQEYSYRO/zvtV/+t3/KkQ4Xjezg6GaljpJCCA1uUujE4Ho8/7u/Hae+9lVua7VtwQDhMs4oKM3hALgN3pMggeRmL1fFW+FbBJRGuOf6Iurzvr5X9jo279mAflSR7COvDfQJVe8LEhnikoixcUwPDPERKN8mWx2qx8ys7yUQxA7/lwjIjpBAYUsaekwUAPkSMPggLSlcrkmb5tJNYaRANGEkNarzz/T+0+T75LMhuNGxH0N2MX8+MAmw0rcjXeainboAWY1/tpeyy3p2OwNpuZm8yfq4Ji8BjwFIugzwJoXY6yRGFSM1TB6+owFLrffGPef3coYAVqbPSNLyUEYxhblTbVJGQbbR9tvuwqnSSk8kr4BSmQK6Hi/9Xz904Ff+33+yazSiZtrptnqr8nDKBzAxviRelK6eiWd+6zejcf1aNCBoy1fOUWi64lCXziWF9UrNRqd6ZC/raAohNsCT9CkkhuvJOLLZQwSXhEqkqtT5sbxpIzR9brsOxFlMUdvF3+GYcu01ZdXlQExOjcXsngnUv2NUQ7lALnuY6fwlTzINwdsWK/FXARkNKhwdWulODcl973kh/1AJmlrLcW2TT7SsE8bnC9gdOqA9JreD5LorXWeiUcuRB+bLrU8Lbc7mcpwPPJJOMcoXgVNmA+RX0IH8hJhPxUi3A8FEbb6/HVMzjD8r0OVrvUonSpfBFRuu3vRBEcwegZY8KBpF7ZTF5KY83+y41C3QTMOl3PpAN3rjcqbzXoVOPvAL/+aPdkW6Pd2BKd8LaTOaeBP1zlBURwZi9eST8dT/87dQ0ws59gRveggvgrTQZLiNSyKKKTGC8i/RLlGWLDE9QjKtP6RNhkpZ71wN0D/PxliGx73zv3mv+B3eL/fcbZcjvWN1j+mp+BcuN8m3ieBvmcZJVCdeTSdR/SEBB+HSOQXUgtHPvJcR5HZZrAYffKOv2iO/BUOn0ZfKvNLHn7RIm+du1tc/dssWcFrSlXOF59CLB6ZPdzzzlM6XOMtrXOGCSkD6NWGarzrg8bOpu1iXwUFptZMNpW9GS1Jm+qD9ev1n2Umz9Ql8yhWoXvO2wxBZrXVCg6tDcvhDYslrPgU88PO//Ye2nH8FTIZ4tqyLN1fleGywFZe/8Vfx9d//9xBcRm1dZuHaJysz/CRHMiHNkoT1iEx8UVFhYrn31zZOk5xsRB887pTQa1z/t9wroL3x3PsFTJb/1jk/mcbrnL1Zc/Zwrgss/aziT5VrSTNaSbCUOUfahDRlrLPl0ipo7Z35hDOl9sstQwBl69N24/YWXTek8z9OLUkSUltQfi6a4zjNCb/9LIb7UJKAV2uqiV2W4q+gFvw6IZmKPGpvH6C0IrHeBCgJADs0hWo1ChHmLWZNbWV7bYMyM7lAIlh8kw4tjEMHluFu2qRfZt84BpHTDICqSgVVUD20tRxzr76Iai3rWdKckNO10/Z82ZmVSITnSUSpVXOVYx+yq0/IDfdLS1Szmjaoya3cM13/t3/cJ7y/9csxf//4rfKts5hYwxwMRe4dHV/C8E2cY+ILzLqdo4Oz3I4tOwnXthr4OVxrtYxsnbhWGApI0GEqOJZlaQZ6e5/OG+nrH7/V3re2G6+ZzFMF54l8M2/hte3RLdC/6ftPdo/SRt8h4IelNW2Og6U55a9CFGySfgRt+cUssgkq2m0w6kOXpaOXzTrdSudUuhyzO5jpMiWRUoYSShvyj1/LGVTlqyZL5gDJ5ZFov1i5O7gVzaWLsTl3nt7QiHWcSsd4TNt3skvDS69xiMHeVJhqGwBnj1DB4rlMcZOIQvaNxP+nm4T+5+79/7shBqqRqfRGBYNvURgoU7zeo0dS4LdaR8kY0xZW9vNzmDGx+fLkhv0/v/VB9TfboUy8ZaBS+lLhiSJMeXHi2JD3fL+3UzTlcgFagpgLmmEX2vkanJqDqAqWPpy+MFWq9bUqyiJpyXy0WPRmRf70Ksy9bJmWf95RicG1PPe376smLWDADZqoEAY5TqHf6Wt7jTw0zkMDO7F+7UwM4BPYU12j5CdGOzRKpDpTbDFW5it8i5kTqTLOXqEzX3qTJPTv5ZZpShSl2ZAef//mpqeUPZTjlGc/f28rZZftxuO/tnFd8Pw1ALEnr9iFjSbBo0xjOfCgMFPWFcfZdKWOkvfGzet/c7d9/t649e9ZtiDpd0allO/b5CgFpZoRPEixzK+hRQBNDdDkCDhpq+QV+EaS2wivye4yZtMaPI1UDTqINJFDUu192pAj1tbDPZeZmEZarNe9LxN9X3WgPnxaDniS+EsWUQ+Xknb23W43/n/2oUxK7SIFjQAAAABJRU5ErkJggg==",
```

Comment:
Hi @khanganbb11,

Could you please update the logo? It currently resembles Kung Fu Panda, which is a trademark of DreamWorks. We need to avoid potential copyright issues. 

Thank you!